Project/backend/codebase/tests/test_json_to_graphml.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import networkx as nx
+from ..graphCreator import json_to_graphml
```

Comment:
Maybe the path of import should be different having in mind that the base cwd would be codebase. Try out the tests locally and see if they pass. If the import is not causing problems you can leave it as it is.

Reply:
The import path works when running `make test` from the backend folder.